format

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -eu
+
+echo "=> Running black"
+black src
+
+echo "=> Running isort"
+isort -q src
+
+echo "=> Running docformatter"
+(cd src && docformatter . || test $$? -eq 3)
+
+echo "=> Running jq"
+for a in $(find src -type f -name "*.jsonschema"); do
+  b=$(jq -S . $a) && echo "$b" >$a || (echo "    in $a"; false)
+done
```

Comment:
In addition to factoring existing bash logic out of `Makefile`, the `jq` formatting step will now report _which file_ had a syntax error. This was fairly obvious before when we only had one or two files, but it's crucial now that we have a growing collection.

Having this logic in its own scripts also means that editors can do better syntax highlighting than when it lived in a `Makefile`.